libraries/psio/include/psio/graphql.hpp

```diff
@@ -359,13 +359,16 @@ namespace psio
                              bool                                        is_query_root)
    {
       (fill_gql_schema((T*)nullptr, stream, defined_types, is_input, false), ...);
```

Comment:
This should probably be inside the condition.  It could cause infinite recursion in some corner cases.

Reply:
Fixed